src/js/modules/infragistics.ui.videoplayer.js

```diff
@@ -49,223 +49,1826 @@
-				/* type="bool" Get or set whether to automatically hide the banner. If set to true, the banner is hidden after banner.hidedelay in milliseconds. */
+				/* type="bool" Get or set whether to automatically hide the banner. If set to true, the banner is hidden after banner.hidedelay in milliseconds.
+				```
+				//Initialize
+					$(".selector").igVideoPlayer({
+							banners: [{
+								imageUrl: "../Content/images/Banner1.png",
+								link: "http://quince.infragistics.com/",
+								times: [5, 20, 60],
+								visible: false,
+								closeBanner: true,
+								animate: true,
+								autohide: true,
+								hidedelay: 10000,
+								width: "200px",
+								height: "67px"
+							}]
+						});
+
+					//Get
+					var banners = $(".selector").igVideoPlayer("option", "banners");
+					var autohide = banners[0].autohide;
+
+					//Set
+					var banners = $(".selector").igVideoPlayer("option", "banners");
+					banners[0].autohide = true;
+				```
+				*/
 				autohide: false,
-				/* type="number" Get or set the banner autohide delay in milliseconds. It is taken into account only if the banner.autohide option is set to true. */
+				/* type="number" Get or set the banner autohide delay in milliseconds. It is taken into account only if the banner.autohide option is set to true.
+				```
+					//Initialize
+					$(".selector").igVideoPlayer({
+							banners: [{
+								imageUrl: "../Content/images/Banner1.png",
+								link: "http://quince.infragistics.com/",
+								times: [5, 20, 60],
+								visible: false,
+								closeBanner: true,
+								animate: true,
+								autohide: true,
+								hidedelay: 10000,
+								width: "200px",
+								height: "67px"
+							}]
+						});
+
+					//Get
+					var banners = $(".selector").igVideoPlayer("option", "banners");
+					var hidedelay = banners[0].hidedelay;
+
+					//Set
+					var banners = $(".selector").igVideoPlayer("option", "banners");
+					banners[0].hidedelay = 10000;
+				```
+				*/
 				hidedelay: 10000,
-				/* type="string" Get or set the banner link that will open in new window. */
+				/* type="string" Get or set the banner link that will open in new window.
+				```
+					//Initialize
+					$(".selector").igVideoPlayer({
+							banners: [{
+								imageUrl: "../Content/images/Banner1.png",
+								link: "http://quince.infragistics.com/",
+								times: [5, 20, 60],
+								visible: false,
+								closeBanner: true,
+								animate: true,
+								autohide: true,
+								hidedelay: 10000,
+								width: "200px",
+								height: "67px"
+							}]
+						});
+
+					//Get
+					var banners = $(".selector").igVideoPlayer("option", "banners");
+					var link = banners[0].link;
+
+					//Set
+					var banners = $(".selector").igVideoPlayer("option", "banners");
+					banners[0].link = "http://quince.infragistics.com/";
+				```
+				*/
 				link: null,
-				/* type="number" Get or set the banner width */
+				/* type="number" Get or set the banner width
```

Comment:
I suppose this can be `number|string`. Not to mention the code snippet shows a string example and the option doc states number.